Server/VikingVault.API/Controllers/AccountsController.cs

```diff
@@ -27,7 +27,10 @@ public AccountsController(IAccountService accService)
         [HttpGet]
         public ActionResult<UserAccount> Get()
         {
-            UserAccount userAccount = _accService.FindById();
+            var token = Request.Headers["x-access-token"];
+
```

Comment:
Remove this empty line. Also, you could  rename _accService into _accountService.